src/command_modules/azure-cli-keyvault/setup.py

```diff
@@ -35,7 +35,7 @@
 
 DEPENDENCIES = [
     'azure-mgmt-keyvault==0.31.0',
-    'azure-keyvault==0.2.0',
+    'azure-keyvault==0.3.0',
```

Comment:
Since Laurent has bumped the version to fix the SSL issue on proxies, could you bump this to 0.3.1?